app/views/planning_applications/neighbour_letters/_letter_template.html.erb

```diff
@@ -37,3 +36,13 @@
   </div>
 </details>
 
+<div class="govuk-hint govuk-!-margin-bottom-0 govuk-!-margin-top-8">Step 4</div>
+<div class="govuk-form-group">
+  <%= form.label :deadline, "Set response period", class: "govuk-label govuk-label--m" %>
+  <p class="govuk-hint">Enter the number of days that neighbours have to respond.</p>
+
+ <div class="govuk-input__wrapper">
+    <%= form.govuk_text_field :deadline, placeholder: "", class: "govuk-input", label: -> {}, value: 21, required: true %>
+    <div class="govuk-input__suffix" aria-hidden="true">days</div>
+  </div>
+</div>
```

Comment:
I'm going to be adding very similar html so I may turn this into a component so we can reuse easily

Reply:
I wonder if the govuk-frontend component library that Andrew linked to the other day might already have one: https://govuk-components.netlify.app/

Reply:
It's in the form builder:
https://govuk-form-builder.netlify.app/form-elements/text-input/#inputs-with-prefixes-and-suffixes